hrm-service/package-lock.json

```diff
@@ -1,13541 +1,590 @@
 {
   "name": "hrm",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
This change again due to npm u_u I think I'll just keep using v1 to avoid introducing this in the future.